extensions/identity-hub/src/test/java/org/eclipse/dataspaceconnector/identityhub/api/IdentityHubControllerTest.java

```diff
@@ -159,6 +159,18 @@ private List<VerifiableCredential> queryVerifiableCredentials() {
                 .body("replies", hasSize(1))
                 .body("replies[0].status.code", equalTo(200))
                 .body("replies[0].status.detail", equalTo("The message was successfully processed"))
-                .extract().body().jsonPath().getList("replies[0].entries", VerifiableCredential.class);
+                .extract().body().jsonPath().getList("replies[0].entries", SampleObject.class);
+    }
+
+    private static class SampleObject {
```

Comment:
why this rather than use VerifiableCredential?

Reply:
It was like this because before we were storing objects and not only verifiable credentials.
Removed it.